### ISSUE TITLE
"403 must be connected to gab"

### ISSUE THREAD

**Issue by force200** [gab-ai-inc/gab-dissenter-extension#87]
Every time I try to comment using Dissenter the error message "403: Failed to Dissient must be connected to gab" pops up. The extension shows that I'm loged in and at closer examination I found out that the bottom of the Extension's window shows "connecting..." that doesn't go away, even If I let it sit for an hour. (As seen in the attached screenshots

I'm using firefox and have AdblockPlus, Ghostry Avira Browser Safety and Duck Duck Go Privacy Essentials, as well a a few other add-ons that are unlikely to be the cause of the issue, installed.
![issue](https://user-images.githubusercontent.com/20172043/59162449-a31f4e00-8af1-11e9-9cd4-d52f17a0943b.png)
![issue2](https://user-images.githubusercontent.com/20172043/59162480-1923b500-8af2-11e9-80e7-3fa6f0350b29.png)

**Comment by justagoodperson** [comment]
I get this error randomly it appears sometimes too. Is it persistent or were you finally able to post at a certain point?

**Comment by force200** [comment]
It happens every time I try to use Dissenter

**Comment by justagoodperson** [comment]
Do you mind to try it out on another browser? At least then we can narrow it down to your specific browser if it works on the other one. I don't have access to the Gab backend so I can't give any insight onto the 403 error but trying another browser seems like a logical first step.

**Comment by force200** [comment]
Yes I do midn. I'd rather not fill my computer with google's spyware!

**Comment by force200** [comment]
It went away on it's own.

**Comment by force200** [comment]
And came back after I restarted the computer.

**Comment by force200** [comment]
Tested it with Microsoft Edge and it seems to work.
![Untitled](https://user-images.githubusercontent.com/20172043/59205179-a3c3ed00-8ba2-11e9-9563-3484976baee3.png)

**Comment by rugabunda** [comment]
@force200
No problem for a week or more, then I had this problem in firefox immediately after I added ghostery, after disabling ghostery it immediately went away, will follow up if anything changes. 

Here: https://dissenter.com/feedback Ghostery block count claimed to be blocking two trackers on the dissenter feedback page, but only one tracker displayed in ghosterys list of blocked trackers. perhaps it was blocking some part of gab

**Comment by rugabunda** [comment]
@force200 it appears to be working fine for 48 hours since disabling ghostery.
